disk/datadog_checks/disk/disk.py

```diff
@@ -357,3 +364,23 @@ def _get_devices_label(self):
             self.log.debug("Couldn't use blkid to have device labels")
 
         return devices_label
+
+    def _get_devices_label_from_blkid_cache(self):
+        devices_label = {}
+        try:
+            with open(self._blkid_cache_file, 'r') as blkid_cache_file_handler:
+                blkid_cache_data = blkid_cache_file_handler.readlines()
+        except IOError as e:
+            self.log.warning("Couldn't read the blkid cache file %s: %s", self._blkid_cache_file, e)
+            return devices_label
+
+        # Line sample
+        # <device DEVNO="0x0801" LABEL="MYLABEL" UUID="..." TYPE="ext4">/dev/sda1</device>
+        for line in blkid_cache_data:
+            root = xml.etree.ElementTree.fromstring(line)
+            device = root.text
+            label = root.attrib.get('LABEL')
```

Comment:
I think we can wrap those 3 lines in a `try/except Exception + log.warning` to catch any parsing error. That will avoid the check run to be stopped on this case.